src/Lti/Message/DeepLinking/LineItem.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Cerpus\EdlibResourceKit\Lti\Message\DeepLinking;
+
+final readonly class LineItem
```

Comment:
Message objects should be extendable and not readonly. It may be desirable to add additional stuff, as is done with LtiLinkItem.